opennlp-tools/src/test/java/opennlp/tools/sentdetect/AbstractSentenceDetectorTest.java

```diff
@@ -30,13 +30,16 @@
 
 public abstract class AbstractSentenceDetectorTest {
 
-  protected static final Locale LOCALE_SPANISH = new Locale("es");
+  protected static final Locale LOCALE_DUTCH = new Locale("nl");
   protected static final Locale LOCALE_POLISH = new Locale("pl");
   protected static final Locale LOCALE_PORTUGUESE = new Locale("pt");
+  protected static final Locale LOCALE_SPANISH = new Locale("es");
```

Comment:
The small details that normally pass unnoticed! Thanks for sorting alphabetically :clap: